nmostesting/NMOSUtils.py

```diff
@@ -57,6 +57,19 @@
     (63072000, 63072009),  # 1 Jan 1972, 10 leap seconds
 ]
 
+DEFAULT_ARGS = {
```

Comment:
Can you reference this object in the setup of the command line arguments? Otherwise the defaults are in danger of getting out of sync.